js/widgets/pitchstaircase.js

```diff
@@ -608,6 +607,11 @@ class PitchStaircase {
         this.widgetWindow = widgetWindow;
         widgetWindow.clear();
         widgetWindow.show();
+        widgetWindow.onclose = () => {
+            logo.synth.setMasterVolume(0);
+            logo.synth.stop();
```

Comment:
No, the `stop()` function does not work at all. It shows in the debug logs that this does not work. So in many widgets like in the temperament and the timbre widget, the `setMasterVolume(0)` is called so as to mute the sounds. However, it does not stop the already started `trigger()` method. Thus, even if we comment out or delete the `logo.synth.stop();` line, it would not have any effect on this functionality.

Reply:
Now, in the function `stop()` of the file synthutils.js, the line `Tone.Transport.stop();` does not work.

Reply:
I investigated. It won't work because `.stop()` stops `Tone.Transport` assignments. Since, notes are triggered by synth's (Tone.js) `triggerAttackRelease` it needs to be released by a `triggerRelease` itself. Since, there are inconsistencies all around, I made a hack fix to this.